src/commands/game_options/duration.ts

```diff
@@ -94,80 +107,226 @@ export default class DurationCommand implements BaseCommand {
         priority: 110,
     });
 
+    slashCommands = (): Array<Eris.ChatInputApplicationCommandStructure> => [
+        {
+            name: "duration",
+            description: LocalizationManager.localizer.translate(
+                LocaleType.EN,
+                "command.duration.help.description"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: DurationActionInternal.SET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.duration.help.example.set"
+                    ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes
+                        .SUB_COMMAND,
+                    options: [
+                        {
+                            name: "duration",
+                            description:
+                                LocalizationManager.localizer.translate(
+                                    LocaleType.EN,
+                                    "command.duration.help.example.set"
+                                ),
+                            required: true,
+                            min_value: DURATION_DELTA_MIN,
+                            max_value: DURATION_DELTA_MAX,
+                            type: Eris.Constants.ApplicationCommandOptionTypes
+                                .INTEGER,
+                        } as any,
+                    ],
+                },
+                {
+                    name: DurationActionInternal.ADD,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.duration.help.example.increment"
+                    ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes
+                        .SUB_COMMAND,
+                    options: [
+                        {
+                            name: "duration",
+                            description:
+                                LocalizationManager.localizer.translate(
+                                    LocaleType.EN,
+                                    "command.duration.help.example.increment"
+                                ),
+                            required: true,
+                            min_value: DURATION_DELTA_MIN,
+                            max_value: DURATION_DELTA_MAX,
+                            type: Eris.Constants.ApplicationCommandOptionTypes
+                                .INTEGER,
+                        },
```

Comment:
Will this need a cast to `any`?

Reply:
Yes until we update Eris to fix their typings